helm/cas-bciers/templates/cronJobs/check-file-status.yaml

```diff
@@ -34,14 +34,17 @@ spec:
               - -c
               - |
                 set -euo pipefail;
-                flock -w 45 --verbose /data/locks/.check.lock \
-                .venv/bin/python manage.py check_document_file_status
+                touch /data/log/cron.log
+                (crontab -l ; echo "* * * * * flock -w 45 --verbose /data/locks/.check.lock .venv/bin/python manage.py check_document_file_status >> /data/log/cron.log 2>&1") | crontab
+                cron && tail -f /var/log/cron.log
```

Comment:
Yo dawg, I heard you liked cronjobs, so I put a cronjob inside your cronjob ;)

👌 This just avoids having this job count against our long running quota.
The only thing is that we let the pod get killed by the cluster instead of shutting down gracefully, so we'll get alerts and failed jobs as part of our normal operation.

we might want to do something like:
```bash
timeout -p 1799s tail -f /var/log/cron.log
```
to keep the job alive until we exit with a zero exit code